src/Graph12/Graph12.tsx

```diff
@@ -0,0 +1,53 @@
+import { ProgressCircle } from "../components/ProgressCircle";
+const data = [
+	{
+		label: "MAGNA",
+
+		count: 878,
+	},
+	{
+		label: "VELIT",
+
+		count: 618,
+	},
+	{
+		label: "DOLOR",
+
+		count: 521,
+	},
+	{
+		label: "CUPLA",
+
+		count: 221,
+	},
+];
+const maxData = 1000;
```

Comment:
Please use uppercase for static values :)

Reply:
You can define this value directly inside <ProgressCircle /> or do something like <ProgressCircle max={MAX_DATA} />

Reply:
done